sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventDataForm.java

```diff
@@ -155,6 +158,15 @@ protected void addFields() {
 		initEventDateValidation(startDate, endDate, multiDayCheckbox);
 
 		addField(EventDto.EVENT_STATUS, OptionGroup.class);
+		addField(EventDto.EVENT_INVESTIGATION_STATUS, OptionGroup.class);
```

Comment:
Since we need to remove the requirement, please use the new NullableOptionGroup here